src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version = "1.0" encoding = "utf-8"?>
+<!DOCTYPE hibernate-configuration SYSTEM
+        "http://www.hibernate.org/dtd/hibernate-configuration-3.0.dtd">
+<hibernate-configuration>
+    <session-factory>
+
+        <property name = "hibernate.dialect">org.hibernate.dialect.MySQL5Dialect</property>
+        <property name = "hibernate.connection.driver_class">com.mysql.jdbc.Driver</property>
+        <property name = "hibernate.connection.url">jdbc:mysql://localhost/cinema?serverTimezone=UTC</property>
+        <property name = "hibernate.connection.username">root</property>
+        <property name = "hibernate.connection.password">1234</property>
+        <property name="show_sql">true</property>
+        <property name="hbm2ddl.auto">create-drop</property>
+
+        <mapping class="ua.com.conductor.model.Movie"/>
+
+    </session-factory>
+</hibernate-configuration>
```

Comment:
is there an empty line here?

Reply:
as for me, if I have blank space it will be more readable